### PR TITLE
[Azure AD] Auto redemption and Cross-tenant sync policy to v1.0

### PR DESCRIPTION
**Pull request by rolyon**

### CODE REVIEW

File: api-reference/v1.0/api/crosstenantidentitysyncpolicypartner-get.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /policies/crossTenantAccessPolicy/partners/{id}/identitySynchronization
+```
+
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/crosstenantidentitysyncpolicypartner-get.md:39]: GET operations should contain the Optional query parameters to describe whether ODATA query parameters are supported or not. Please, include the Optional query parameters section to this topic. For an example, see https://learn.microsoft.com/en-us/graph/api/learningprovider-get?view=graph-rest-1.0&tabs=http#optional-query-parameters.

Same here: 
- api-reference/beta/api/crosstenantidentitysyncpolicypartner-get.md

**Comment by JarbasHorst** [comment]
Hello @rolyon, I reviewed your PR and left one comment. Please, take a look when you will have time.

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -9690,25 +9748,25 @@
         {
           "Id": "c71b4a31-ce26-4317-8dbc-ec6aa654d9e4",
           "ApiChange": "Resource",
-          "ChangedApiName": "privilegedapproval",
+          "ChangedApiName": "privilegedApproval",
```
Comment (rolyon) [inline changelog/Microsoft.DirectoryServices.json:9751]: @JarbasHorst
Please remove these changes. These changes are outside the scope of this pull request.
thanks

Comment (JarbasHorst) [reply changelog/Microsoft.DirectoryServices.json:9751]: Hi, these changes are needed because they were causing invalid link validation. If we will revert these changes, your PR will not pass the following validation:

image

Comment (FaithOmbongi) [reply changelog/Microsoft.DirectoryServices.json:9751]: Hi @Jarbas-MSFT , @rolyon - I've pushed a separate commit to resolve this changelog error because many PRs are affected. Unfortunately, you'll get a merge conflict on your PR.

File: api-reference/v1.0/api/crosstenantidentitysyncpolicypartner-get.md (hunk 1 above)
Comment (rolyon) [reply api-reference/v1.0/api/crosstenantidentitysyncpolicypartner-get.md:39]: See https://github.com/microsoftgraph/microsoft-graph-docs/pull/21010/commits/2053565c6c8bf6acc3b0accbd777133e9b42a262

**Comment by JarbasHorst** [comment]
Hi @Lauragra, this PR is ready for final review. 

Hi @FaithOmbongi, can you please verify whether we still need the label "blocked"? Thanks :)

**Review by FaithOmbongi — APPROVED** [review]
Changes look great, thanks @Jarbas-MSFT and @rolyon!

@rolyon - You can remove the blocked and do not merge labels when you're ready for the PR to be merged.
